function-support.html

```diff
@@ -178,8 +178,22 @@
   color: inherit;
 }
 
-#toc li ul         { display: none; }      /* default is closed */
-#toc li.li-open ul { display: block; }     /* opened by a click */
+#toc input[type=checkbox] {
+	/* hide native checkbox */
+	position: absolute;
+	opacity: 0;
+}
+
+#toc ul {
+	display: none;     /* by default hide all sub menus */
+}
+#toc label::after {
+   content: ' ▼';

```

Comment:
It's a little weird to have a mix of spaces and tabs for indents, but I'm not too concerned about that for this file.